lib/web/apiserver.go

```diff
@@ -3435,6 +3437,8 @@ func (h *Handler) clusterUnifiedResourcesGet(w http.ResponseWriter, request *htt
 
 	getUserGroupLookup := h.getUserGroupLookup(request.Context(), clt)
 
+	allComponentFeatures := h.getAllClusterComponentFeatures(request.Context())
```

Comment:
Does allComponents means also db/app agent ?  I think that this should be more explict with desc why proxy and auth features here are relevant